bin/rpm-s3

```diff
@@ -214,17 +214,8 @@ def update_repodata(repopath, rpmfiles, options):
         # don't put a base url in <location> tags of primary.xml.gz
         newpkg._baseurl = None
         older_pkgs = yumbase.pkgSack.searchNevra(name=newpkg.name)
+        new_packages.extend(older_pkgs)
```

Comment:
Do we need to do that? From what I understand, by doing this the older packages would be listed twice (since we have:

```
mdconf.pkglist = list(yumbase.pkgSack) + new_packages
```
below, with `yumbase.pkgSack` already containing the old packages).

I think we also want to keep
```
        for i, older in enumerate(reversed(older_pkgs), 1):	
            if older.pkgtup == newpkg.pkgtup:	
                yumbase.pkgSack.delPackage(older)	
                logging.info('ignoring: %s', older.ui_nevra)

        if options.delete is False:	
            new_packages.append(newpkg)
```

or an equivalent logic, as we won't be able to remove packages with `rpm-s3` without it.

Reply:
For some reason my brain though `keep` and `delete` were the same. Fixed.